database/src/main/scala/cromwell/database/slick/SlickDatabase.scala

```diff
@@ -602,7 +602,7 @@ class SlickDatabase(databaseConfig: Config) extends SqlDatabase {
     runTransaction(action)
   }
 
-  protected def addMetadata(events: Iterable[Metadatum])
+  override def addMetadata(events: Iterable[Metadatum])
```

Comment:
Yeah so I asked myself the same question - but the ticket said move out the "business logic" code to their respective sub-projects. Which I interpreted as all the raw Database query implementations can stay in the same SlickDataAccess file in the database, and the things with custom complex types on top of this have to move. But I might have misunderstood maybe we want to split SlickDataAccess too..